config/kraft/controller.properties

```diff
@@ -26,13 +26,13 @@ process.roles=controller
 # The node id associated with this instance's roles
 node.id=1
 
-# The connect string for the controller quorum
-controller.quorum.voters=1@localhost:9093
+# List of controller endpoints used connect to the controller cluster
+controller.quorum.bootstrap.servers=localhost:9093
 
 ############################# Socket Server Settings #############################
 
 # The address the socket server listens on.
-# Note that only the controller listeners are allowed here when `process.roles=controller`, and this listener should be consistent with `controller.quorum.voters` value.
+# Note that only the controller listeners are allowed here when `process.roles=controller`
```

Comment:
does this listener still need to be consistent with what `controller.quorum.bootstrap.servers` contains?

Reply:
In most cases yes but it is not required. The best example is that the controller.quorum.boostrap.servers could be a domain name that resolves to any of the controller hosts.